addons/default/visiosoft/profile-module/src/Http/Controller/MyProfileController.php

```diff
@@ -272,7 +275,18 @@ public function getEducation(Request $request)
     public function changeEducation(Request $request)
     {
         if ($request->info == 'education') {
-            $education = EducationPartModel::query()->where('education_id', $request->education)->get();
+            $education = DB::table('profile_education_part')
+                ->leftJoin(
+                    'profile_education_part_translations',
+                    'profile_education_part.id',
+                    '=',
+                    'profile_education_part_translations.entry_id'
+                )
+                ->where('profile_education_part_translations.locale', '=', Request()->session()->get('_locale', setting_value('streams::default_locale')))
+                ->where('education_id', $request->education)
+                ->selectRaw('default_profile_education_part.*, default_profile_education_part_translations.name as name')
+                ->orderBy('name', 'ASC')
+                ->get();
```

Comment:
why left join?

Reply:
for order by name

Reply:
can't you just do EducationPartModel::query()->where('education_id', $request->education)->get()->sortBy('name'); ?